### ISSUE TITLE
Initial Documentation

### ISSUE THREAD

**Issue by sebastianburckhardt** [Microsoft/ReactiveMachine#4]
We need a presentable first version of the docs.

**Comment by MedAnd** [comment]
Please consider documenting the comparison of ReactiveMachine vs Ambrosia. After reading the current docs on both sites I'm not sure if I want both, one or the other.

**Comment by sebastianburckhardt** [comment]
Good point. I will add more information to the docs (which are still under heavy editing).
In the meantime I have added clarifying comment in #3.
